examples/multi_arch_and_repo/test.yaml

```diff
@@ -4,7 +4,7 @@ commandTests:
     command: "curl"
     args: ["--version"]
     expectedOutput:
-      - curl 8.4.0
+      - curl 8[.].*
```

Comment:
we should keep this static to assert that we are really reproducible.

Reply:
Fixed the version. 
[agree as soon as we will be hermetic].